tornado/test/netutil_test.py

```diff
@@ -200,3 +200,13 @@ def test_same_port_allocation(self):
         finally:
             for sock in sockets:
                 sock.close()
+
+    def test_reuse_port(self):
+        port = get_unused_port()
```

Comment:
`get_unused_port()` is flaky, especially on travis-ci, because the port may become used by another process on the same machine before we can grab it. Instead, this test should use bind_unused_port() (adding a `reuse_port` argument to that function too), and then try to bind a second time with `bind_sockets(reuse_port=True)`. Also, the test should be skipped whenever `reuse_port` is not supported.